pkg/apis/options/options.go

```diff
@@ -2,9 +2,9 @@ package options
 
 import (
 	"crypto"
+	internaloidc "github.com/oauth2-proxy/oauth2-proxy/v7/pkg/oidc"
```

Comment:
This shouldn't be here, it should be imported inline with the other oauth2-proxy imports below. I think the linter should be complaining about this